web/app/js/components/MetricsTable.jsx

```diff
@@ -142,6 +151,11 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
     }
   ];
 
+  // don't add the meshed column on a Authority MetricsTable
+  if (resource !== "authority") {
```

Comment:
I feel like there is a better way to handle this case and not have to single out the `Authority` resource to exclude the meshed column from it. Any feedback is appreciated.

Reply:
I think this is fine!